backend/compact-connect/stacks/persistent_stack/__init__.py

```diff
@@ -109,16 +119,13 @@ def __init__(
 
         # This resource should not be referenced directly as a cross stack reference, any reference should
         # be made through the SSM parameter
-        self._data_event_bus = EventBus(self, 'DataEventBus')
+        self._data_event_bus = EventBus(self, 'DataEventBus', event_bus_name=f'{environment_name}-dataEventBus')
```

Comment:
Let's add a note here that changing this name would be a breaking interface change that would require a blue-green migration strategy to move consumers over.